Formula/pgweb.rb

```diff
@@ -15,21 +15,15 @@ class Pgweb < Formula
   end
 
   depends_on "go" => :build
-  depends_on "go-bindata" => :build
 
   def install
-    ENV["GOPATH"] = buildpath
-    ENV["GO111MODULE"] = "auto"
-    (buildpath/"src/github.com/sosedoff/pgweb").install buildpath.children
+    ldflags = %W[
+      -s -w
+      -X github.com/sosedoff/pgweb/pkg/command.BuildTime=#{time.iso8601}
+      -X github.com/sosedoff/pgweb/pkg/command.GoVersion=#{version}
```

Comment:
Shouldn't this be the version of the Go runtime being used in build, not the version of `pgweb`?

There's also a variable `GitCommit`, not sure if that needs to be set.

Reply:
yeah, I am aware of that GitCommit thing, but I dont know why that is useful (I feel the version and builtBy would be enough)

Reply:
Okay. But regarding `GoVersion`, maybe that should be `Formula["go"].version` or something like that?

Reply:
you are absolutely right, I dont know why upstream is using GoVersion, let me file a PR asking about it. 😂

Reply:
I see, https://github.com/sosedoff/pgweb/blob/4de356cb1d47d7407169d75a710520c6cd184fb7/pkg/command/version.go#L4-L5, my bad.